src/helpers/GCNotifyCaller.ts

```diff
@@ -156,4 +156,99 @@ export default class GCNotifyCaller {
+        let phoneResponse;
+        let emailResponse;
+
+        // Attempt to send text x number of times. Throw error otherwise
+        for (let i = 0; i < this.retryLimit; i++) {
+            if (personalisation && Object.hasOwn(personalisation, 'pin')) {
+                (personalisation as any).pin =
+                    ((personalisation as any).pin as string).substring(0, 4) +
+                    '-' +
+                    ((personalisation as any).pin as string).substring(4);
+            }
+
+            try {
+                // eslint-disable-next-line @typescript-eslint/no-unused-vars
+                emailResponse = await this.sendEmail(
+                    notifyClient,
+                    emailTemplateId,
+                    email,
+                    { personalisation: personalisation },
+                );
+
+                // eslint-disable-next-line @typescript-eslint/no-unused-vars
+                phoneResponse = await this.sendSms(
+                    notifyClient,
+                    phoneTemplateId,
+                    phone,
+                    { personalisation: personalisation },
+                );
+
+                if (emailResponse | phoneResponse) {
+                    return true;
+                }
+            } catch (err) {
+                if (emailResponse) {
+                    return true;
+                } else {
+                    try {
+                        if (!phoneResponse) {
+                            // eslint-disable-next-line @typescript-eslint/no-unused-vars
+                            phoneResponse = await this.sendSms(
+                                notifyClient,
+                                phoneTemplateId,
+                                phone,
+                                { personalisation: personalisation },
+                            );
+                        }
+                        return emailResponse | phoneResponse ? true : false;
+                    } catch {
+                        let message =
+                            `Error(s) sending GCNotify email & text - ` +
+                            (err as gcNotifyError).response.status +
+                            ` ` +
+                            (err as gcNotifyError).response.statusText +
+                            `:`;
+                        for (const error of (err as gcNotifyError).response.data
+                            .errors) {
+                            message =
+                                message +
+                                `\n` +
+                                error.error +
+                                `: ` +
+                                error.message;
+                        }
+                        logger.error(message);
+                        if (i + 1 === this.retryLimit) {
+                            throw new Error(message);
+                        }
+
+                        return emailResponse | phoneResponse ? true : false;
```

Comment:
Super picky again, but this specifically should be a continue statement. That's because if you get here, you will have failed both email and the phone notification, and we want to try again until the retry limit, which throws an error in the statement above.